lib/formtools/plugins/plugins-helpers.js

```diff
@@ -252,6 +242,50 @@ var filtersDefault = function (filters) {
 
 };
 
+var exportDefault = function (exports) {
+
+    if (exports === undefined || exports === null) {
+        return undefined;
```

Comment:
I see, no wonder I had to merge mine with the latest from your remote branch. I deleted my local version and pull yours down again as a fresh branch to avoid the unnecessary merge :smile: